ddtrace/contrib/redis/util.py

```diff
@@ -41,7 +41,10 @@ def format_command_args(args):
     out = []
     for arg in args:
         try:
-            cmd = stringify(arg)
+            if isinstance(arg, bytes):
```

Comment:
I think we should leave this in place and try to decode and pass the args in aioredis instead of doing this for all the redis integrations.

Also, the above docstring is out of date with this change too... `- Skip binary content`